tests/src/CLRTest.Execute.Bash.targets

```diff
@@ -356,8 +359,20 @@ ReleaseLock()
         return 3
     fi
 }
+
+CheckForCrossgenMarker()
+{
+    echo "in CheckForCrossgenMarker"
+    if [ ! -f "$MarkerFile" ]%3B then
+        sleep 1
+        CheckForCrossgenMarker
```

Comment:
Unbounded recursion seems unnecessary.  Wouldn't a simple `for` or `while` loop be better.